src/types/module.ts

```diff
@@ -64,6 +64,40 @@ export interface UploadcareOptions {
   [key: string]: any
 }
 
+export interface IPXModifiers extends ImageModifiers {
+  format: 'jpeg' | 'jpg' | 'png' | 'webp' | 'avif' | 'gif' | 'heif' | 'tiff' | 'auto' | string
+  fit: 'contain' | 'cover' | 'fill' | 'inside' | 'outside' | string
+  resize: string
+  quality: number | string
+  background: string
+  position: string
+  enlarge: true | 'true'
+  kernel: 'nearest' | 'cubic' | 'mitchell' | 'lanczos2' | 'lanczos3' | string
+  trim: number | string
+  extend: string
+  extract: string
+  rotate: number | string
+  flip: true | 'true'
+  flop: true | 'true'
+  sharpen: number | string
+  median: number | string
+  blur: number | string
+  flatten: true | 'true'
+  gamma: string
+  negate: true | 'true'
+  normalize: true | 'true'
+  threshold: number | string
+  modulate: string
+  tint: number | string
+  grayscale: true | 'true'
+  animated: true | 'true'
+}
+
+export interface StaticOptions extends Partial<IPXOptions> {
+  baseURL: string
+  modifiers: Partial<IPXModifiers>
```

Comment:
Would be nice to rename it to `StaticIPXOptions` also i think `baseURL` at least is shared also for runtime with v2 migration